Sources/Sentry/Transactions/SentrySpanContext.m

```diff
@@ -0,0 +1,59 @@
+//
+//  SpanContext.m
+//  Sentry
+//
+//  Created by Dhiogo Brustolin on 05/01/21.
+//  Copyright © 2021 Sentry. All rights reserved.
+//
+
+#import "SentrySpanContext.h"
+#import "SentryId.h"
+#import "SentrySpanId.h"
+
+@interface
+SentrySpanContext () {
+    NSMutableDictionary<NSString *, NSString *> *_tags;
+}
```

Comment:
`l`: I think we could remove this and also `- (NSMutableDictionary *)tags`. We could just allocate the dictionary on line 47, or am I missing something?